proposals/description/fip_73_swap.ts

```diff
@@ -0,0 +1,81 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_73: ProposalDescription = {
+  title: 'FIP-73: Fei Peg Policy Changes',
+  commands: [
+    {
+      target: 'bammDeposit',
+      values: '0',
+      method: 'withdraw(address,uint256)',
+      arguments: ['{feiDAOTimelock}', '20000000000000000000000000'],
+      description: 'Withdraw 20M LUSD from BAMM to the DAO Timelock'
+    },
+    {
+      target: 'lusd',
+      values: '0',
+      method: 'approve(address,uint256)',
+      arguments: ['{curveLusdMetapool}', '20000000000000000000000000'],
+      description: 'Approve 20M LUSD on Curve LUSD metapool'
+    },
+    {
+      target: 'curveLusdMetapool',
+      values: '0',
+      method: 'exchange_underlying(int128,int128,uint256,uint256,address)',
+      arguments: [
+        '0', // i
+        '1', // j
+        '20000000000000000000000000', // dx
+        '19960000000000000000000000', // min_dy (tolerate 0.2% slip)
+        '{compoundDaiPCVDeposit}'
+      ],
+      description: 'Swap 20M LUSD on Curve for at least 19.96M DAI and send to Compound DAI deposit'
+    },
+    {
+      target: 'compoundDaiPCVDeposit',
+      values: '0',
+      method: 'deposit()',
+      arguments: [],
+      description: 'Deposit 20M DAI on Compound'
```

Comment:
why are we depositing to compound? can't we just send straight to dai psm?

Reply:
it may never go through the dripper / psm if there are no redemptions

Reply:
anyway a permissionless call on the PSM will re-allocate the extra DAI to compound